scripts/leaderboard.coffee

```diff
@@ -164,15 +220,18 @@ module.exports = (robot) ->
         start+=testword.length
       start-=testword.length
 
+      # convert score to rank
+      result.rank = getRank(result.New)
+      result.index = ranks.indexOf(result.rank)
+
       # generates response message for reply
-      newmsg = "[#{result.Response} #{result.Name} now at #{result.New}]"
+      newmsg = "[#{result.Response} #{result.Name} is now at :rank#{result.index}: #{result.rank}]"
```

Comment:
How about sending the `abc is now at xyz rank` message only on rank change, and sending the updated score and current rank in all other cases?

Reply:
I first thought this should be the case but I decided not to go with it because there is no system in place that tracks the rank of the member constantly. The user might forget about the existence of ranks if we don't show them often.

For example, a Bastion III rank would take around 50 points to climb up to the next rank but a Bearer of Excellence II will require 100 points. 

If they don't see the ranks often then they can forget about the rank they are at or the ranking system altogether.